### ISSUE TITLE
FR: Add platform specific Cloud Message attributes

### ISSUE THREAD

**Issue by tsuijten** [firebase/firebase-admin-node#151]
### Environment

  * Operating System version: Node
  * Firebase SDK version: 5.2.1
  * Library version: 5.2.1
  * Firebase Product: Cloud Messaging

### Problem

You can send platform-specific keys as described here: https://firebase.google.com/docs/cloud-messaging/concept-options

However, the SDK doesn't allow this. When I try to send `android` or `apns` specific data with `sendToDevice` I get an error stating this data is not allowed.

#### Relevant Code:

```
    const response = await admin.messaging().sendToDevice(tokens, {
      notification: {
        title: 'Hi',
        body: 'There',
      },
      android: {
        data: {
          custom: 'key',
        }
      },
    });
```

**Comment by hiranya911** [comment]
This is being implemented at #187

**Comment by hiranya911** [comment]
This is available with the new `messaging.send()` method that was released today: https://github.com/firebase/firebase-admin-node/releases/tag/v5.9.0
